requirements-travis.txt

```diff
@@ -16,3 +16,6 @@ psutil==3.1.1
 # stevedore for loading "new style" plugins
 stevedore==1.8.0
 lxml>=3.4.4
+# some setuptools versions can fail accessing
+# pkg_resources.packaging, let's pin the version
+setuptools==25.1.1
```

Comment:
Do you think it's necessary to set this `using >=` for `requirements.txt` as well or is it safe to leave it on platform itself?